lib/pages/login-page.js

```diff
@@ -21,29 +22,32 @@ export default class LoginPage extends AsyncBaseContainer {
 		const userNameSelector = By.css( '#usernameOrEmail' );
 		const passwordSelector = By.css( '#password' );
 
-		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
-		await this.driver.sleep( 1000 );
+		await this.driver.wait(
```

Comment:
If this really does work better than `waitTillPresentAndDisplayed`, then it would probably be better to update that rather than just fixing in this one spot

Reply:
Thanks for double checking on this @bsessions85. I've done additional research and confirm that it is better to use `waitTillPresentAndDisplayed` as it provides more coverage.

- `driver.wait(until.elementLocated)` - checks that an element is _present_ within the HTML DOM of a page. This does not necessarily mean that the element is _visible_. So there's no guarantee that it's _interactable_.

- `waitTillPresentAndDisplayed` function - includes `driver.findElement()` and `element.isDisplayed()` - in other words, it checks that an element is _present_ within the HTML DOM of a page and is _visible_. This does not necessarily mean that the element is _interactable_ or _clickable_ but at least it is already _visible_. 

>Part of the problem with removing sleeps is that they are often times put in on a day when things are running slower or the tests start failing suddenly. A slight delay is added and things start passing again. That being said, it is important to think through what might be happening that caused the sleep to be added in the first place.

`waitTillPresentAndDisplayed` is followed by `setWhenSettable` which finds the element, clears the text from the textarea and enters the value. 

I checked why the sleep statement was added here https://github.com/Automattic/wp-e2e-tests/pull/1528 - tests run too fast and a delay is added. While there is no explanation of why the test is failing exactly, the reason could be that there is no check if the cursor had been placed in the input field or if the input field has a focus. If the test runs too fast, the input field may be disaplyed but not in focus yet while the driver is trying to clear it already. 

I am working on testing my theory above. Will submit another PR as soon as I am done.